samples/net/openthread/coprocessor/overlay-tlsr9-br.conf

```diff
@@ -0,0 +1,7 @@
+# Telink B91 TLSR9518ADK80D
```

Comment:
I think it'd be better to add `boards/tlsr9518adk80d.conf` file to the sample - that way the board-specific configuration would be applied automatically, no need for an overlay.

Reply:
Done.  Created new `boards/tlsr9518adk80d.conf` file with the same content.

Please pay attention that this conf file is NOT loaded automatically.
With this sample I have to run cmake with `-DCONF_FILE="prj.conf overlay-rcp.conf"` and as I see the files from `boards` folder are not loaded automatically if `-DCONF_FILE` option is present.
Now I run with `-DCONF_FILE="prj.conf overlay-rcp.conf boards/tlsr9518adk80d.conf"`

Reply:
If you specify a custom configuration explicitly with `-DCONF_FILE`, then indeed, that configuration takes precedence (by design). But if you operate on defaults (which is what we should aim for the samples), then the config file in boards directory should be merged with `prj.conf` automatically, when built for that board.

If you need to specify an overlay file to build with, please use `-DOVERLAY_CONFIG=overlay-rcp.conf` instead of overriding the entire configuration. In result, all 3 config files should be merged.

Reply:
Yes, it works. 
Thanks.